src/docs/self-hosted/external-storage.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: External Storage

```

Comment:
Maybe we should generalize to "Data Storage" or something.

This way this document can explain where the data is stored by default and can list alternatives if there are any.

Could have the following sections:

- Sentry (with a general explanation about postgres, clickhouse and kafka maybe)
	- Filestore (Uploads, Replays)
		- Database
		- Object Storage
	- Nodestore (Event data)
		- Database
- Vroom (Profiles)
	- Docker volume
	- Object Storage

We should probably either rename those section to what the specifically store or explain that in the intro because "Vroom" is not very descriptive but if it's explained that that component is responsible for (ingest and) storing profiling data it makes a lot more sense.

Maybe with until someone else also chimes in before rewriting the whole thing in case I'm off base with this outline but this sounds like a document I would love to have had when I started my self-hosted adventures 👍